index.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
     "no-concurrent-async": require("./rules/no-concurrent-async"),
     "no-deprecated-components": require("./rules/no-deprecated-components"),
     "no-cyclic-modules-imports": require("./rules/no-cyclic-modules-imports"),
+    "enforce-eslint-mandatory-prop": require("./rules/enforce-eslint-mandatory-prop"),
```

Comment:
i'd probably rename the rule `enforce-optional-prop` we don't need to mention it is eslint one.

Reply:
@tondi i's minor, even you keep the original proposal it's ok.

Reply:
Renamed as suggested